src/messages/messages.json

```diff
@@ -4,14 +4,15 @@
   "finding_coverage_report": "Finding Parasoft coverage XML report...",
   "finding_coverage_report_in_working_directory": "Finding Parasoft coverage XML report in working directory {0}...",
   "found_coverage_report": "Found Parasoft coverage XML report file: {0}",
-  "found_multiple_coverage_report": "Found {0} Parasoft coverage XML report files",
+  "found_multiple_coverage_report": "Found {0} Parasoft coverage XML report paths:",
```

Comment:
Please change to `Found {0} matching files:` because at this point we are not sure whether it's a Parasoft coverage report

Reply:
Please change the message at line 6 too